tests/exploits/cve_2024_6387_test.py

```diff
@@ -0,0 +1,60 @@
+from unittest import mock
+from unittest.mock import patch
+from agent import definitions
+from agent.exploits import cve_2024_6387
+
+
+def create_target(scheme: str, domain: str, port: int) -> definitions.Target:
+    """Helper function to create a Target instance.
+
+    Args:
+        scheme: The URL scheme.
+        domain: The domain or IP address.
+        port: The port number.
+
+    Returns:
+        A Target instance.
+    """
+    return definitions.Target(scheme, domain, port)
```

Comment:
is this really needed ? just inline in the tests:
`target = definitions.Target(scheme="ssh", domain="dummy_vulnerable_ip", port=22)`